cypress/e2e/contactForm.cy.js

```diff
@@ -28,4 +36,28 @@ describe('Contact', () => {
 
     contactForm.assertAllert(testData.successMessage);
   });
+
+  it('should complete the automation flow', () => {
+    homePage.visitHomePage();
+    homePage.clickLaptops();
+    laptopsPage.clickSonyVaioI7();
+    cartPage.clickAddToCart();
+    cartPage.assertProductInCart('Sony vaio i7');
+    cartPage.clickCart();
+    placeOrderPage.clickPlaceOrder();
+    placeOrderPage.fillOrderForm(
+      'John Doe',
+      'United States',
+      'New York',
+      '1234567890123456',
+      '12',
+      '2024'
```

Comment:
You imported faker so use faker. Data should be randomized. 
also
It'd be better to initialize e.g. `user` variable first and you use it here and in `assertEnteredData()` function later on.